proto/nodenetworkservice/3.302.0.744/NodeNetworkService.proto

```diff
@@ -0,0 +1,53 @@
+syntax = "proto3";
+package azure.nodenetworkservice;
```

Comment:
can it be azurenodenetworkservice or azurenns

Reply:
This comes from nns team, do I didn't want to rename it. When we generate go contracts from it, we override this package name to nodenetworkservice.